src/common/snippets/src/lowered/pass/extract_loop_invariants.cpp

```diff
@@ -91,26 +91,28 @@ void update_loop_ports(const ExpressionPtr& expr, const LoopManagerPtr& loop_man
             }
         }
     }
-    const auto& expr_input_ports = expr->get_input_ports();
-    inner_loop_info->update_loop_ports(expr_input_ports, new_loop_input_ports);
+
+    bool inserted = false;
+    std::map<ExpressionPort, std::vector<ExpressionPort>> port_mapping;
+    for (const auto& port : expr->get_input_ports()) {
+        if (!inserted && inner_loop_info->is_loop_port(port)) {
+            port_mapping[port] = new_loop_input_ports;
+            inserted = true;
+        } else {
+            port_mapping[port] = {};
+        }
+    }
+    inner_loop_info->replace_with_new_ports(port_mapping);
```

Comment:
Could you please add a one-line comment to this block? Two other blocks in this function are commented - and it helps a lot.
For example, it's not that straightforward that `port_mapping[port] = {}` means `delete loop port`

Also, `inner_loop_begin_pos` and `inner_loop_end_pos` arguments of this function seems to be redundant. Can you remove them please?

Reply:
Added comment, I hope it's better, thanks you!

> Also, inner_loop_begin_pos and inner_loop_end_pos arguments of this function seems to be redundant. Can you remove them please?

Sure, thanks!